src/generic/graphicc.cpp

```diff
@@ -1124,15 +1130,22 @@ wxCairoFontData::InitFontComponents(const wxString& facename,
 }
 
 wxCairoFontData::wxCairoFontData( wxGraphicsRenderer* renderer, const wxFont &font,
-                         const wxColour& col )
+                                  const wxRealPoint& dpi, const wxColour& col )
     : wxGraphicsObjectRefData(renderer)
 #ifdef __WXGTK__
     , m_wxfont(font)
 #endif
 {
     InitColour(col);
 
+#ifndef __WXMAC__
```

Comment:
I'd prefer to test for `__WXMAC__` rather than its negation, it's more clear and more consistent with the existing tests below.